packages/twenty-server/src/engine/modules/workspace/workspace.resolver.ts

```diff
@@ -113,4 +113,17 @@ export class WorkspaceResolver {
       workspaceId: workspace.id,
     });
   }
+
+  @Mutation(() => String)
+  async removeWorkspaceMember(
+    @AuthWorkspace() { id: workspaceId }: Workspace,
+    @Args('memberId') memberId: string,
+  ) {
+    const workspaceMemberId = await this.workspaceService.removeWorkspaceMember(
+      workspaceId,
+      memberId,
+    );
+
```

Comment:
you need to emit an event `workspaceMember.deleted` in order to update the subscription status
`this.eventEmitter.emit('workspaceMember.created', payload);`
See example in packages/twenty-server/src/engine/modules/user-workspace/user-workspace.service.ts

Reply:
I just changed the event structure in #4566 - will address comments right now to merge it quickly